gateways/clients/wallet_service_db_client.py

```diff
@@ -114,7 +132,7 @@ def get_address_balance(self, address: str, token: str) -> dict:
         return result._asdict()
 
     def get_address_history(
-        self, address: str, token: str, limit: int, offset: int
+        self, address: str, token: str, limit: int, last_tx: str, last_ts: int
```

Comment:
The limit argument is not used on the query anymore.
Previously the frontend had a hardcoded limit of 10 transactions per page so this would not change much, but I think we should be able to search smaller or larger pages, e.g. from 1 to 100 transactions per page.